packages/storybook/src/nimble/select/select.stories.ts

```diff
@@ -245,10 +242,26 @@ export const select: Meta<SelectArgs> = {
         dropDownPosition: 'below',
         appearance: DropdownAppearance.underline,
         optionsType: ExampleOptionsType.simpleOptions,
-        clearable: false
+        clearable: false,
+        loadingVisible: false
     }
 };
 
+export default metadata;
+
+export const select: StoryObj<SelectArgs> = {
+    ...metadata,
```

Comment:
I don't think we have had to do this before. And it runs contrary to my understanding of how metadata works in storybook. i.e. should already expect that metadata is shared between each story. Seems fishy

Reply:
I'll try without it.

Reply:
Removed.